content/app/development/ux/images/_index.en.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Images
+description: How to add and edit image references.
+toc: true
+weight: 40
+---
+
+## Add images to the application
+
+Adding images is done in _FormLayout.json_ by using the image component. Alternative text is useful for scenarios when the image cannot be loaded for any reason, or for users that use screen readers. This can be added as a text resource, defined in _resource.[språk].json_.
```

Comment:
```suggestion
Adding images is done in _FormLayout.json_ by using the image component. Alternative text is useful for scenarios when the image cannot be loaded for any reason, or for users that use screen readers. This can be added as a text resource, defined in _resource.[language].json_.
```